lib/ansible/plugins/callback/actionable.py

```diff
@@ -22,9 +22,31 @@
       - default_callback
     requirements:
       - set as stdout callback in configuration
+    # Override defaults from 'default' callback plugin
+    options:
+      display_skipped_hosts:
+        name: Show skipped hosts
+        description: "Toggle to control displaying skipped task/host results in a task"
+        type: bool
+        default: no
+        env:
+          - name: DISPLAY_SKIPPED_HOSTS
```

Comment:
I imagine this just mirrors the default plugin, but why doesn't the env var start with ANSIBLE?

Reply:
This does reflect the setting from the `default` plugin. The reason for the name is probably lost to history, but it can't be changed without breaking things for people using the old env var name.

Reply:
you can deprecate it and add a new one

Reply:
Turns out it was in commit d5f20e6b2163b7bc74f5855f1da58cabd30edcbd that DISPLAY_SKIPPED_HOSTS was added. At that time, it was the only env var in lib/ansible/constants.py to NOT start with `ANSIBLE_`

Reply:
opened an issue for it: https://github.com/ansible/ansible/issues/52967